system/doc/reference_manual/expressions.xml

```diff
@@ -1390,14 +1390,15 @@ Ei = Value |
       <item>For <c>binary</c> and <c>bitstring</c> it is
       the whole binary or bit string.</item>
     </list>
-    <p>In matching, this default value is only
-    valid for the last element. All other bit string or binary
-    elements in the matching must have a size specification.</p>
+    <p>In matching, the default value for a a binary or bit string
```

Comment:
double a's `the default value for a a binary`.

Reply:
Will fix.